```
> doca-bootstrap-theme@0.2.0 lint /Users/miksu/Projects/doca-bootstrap-theme
> eslint components || true


/Users/miksu/Projects/doca-bootstrap-theme/components/constraints.js
   3:7   error  'ImmutablePropTypes' is assigned a value but never used  no-unused-vars
  12:3   error  render should be placed after considerType               react/sort-comp
  38:68  error  There should be no spaces inside this paren              space-in-parens
  40:13  error  Expected line break before `.reduce`                     newline-per-chained-call
  40:19  error  There should be no spaces inside this paren              space-in-parens
  40:56  error  There should be no spaces inside this paren              space-in-parens
  40:58  error  There should be no space before '}'                      react/jsx-curly-spacing
  52:15  error  Unexpected space before function parentheses             space-before-function-paren
  52:24  error  Block must not be padded by blank lines                  padded-blocks
  54:5   error  Expected space(s) after "if"                             keyword-spacing
  54:26  error  Missing space before opening brace                       space-before-blocks
  55:14  error  Strings must use singlequote                             quotes
  55:14  error  Unexpected string concatenation                          prefer-template
  55:29  error  Strings must use singlequote                             quotes
  57:5   error  Expected space(s) after "if"                             keyword-spacing
  57:24  error  Missing space before opening brace                       space-before-blocks
  58:14  error  Strings must use singlequote                             quotes

✖ 17 problems (17 errors, 0 warnings)

~/Projects/doca-bootstrap-theme: Relequestual-master 
```

--- a/components/constraints.js
+++ b/components/constraints.js
@@ -1,6 +1,5 @@
 const React = require('react');
 const Component = require('react-pure-render/component');
-const ImmutablePropTypes = require('react-immutable-proptypes');
 const _ = require('lodash/core');
 
 class Constraints extends Component {
@@ -8,6 +7,16 @@ class Constraints extends Component {
   static propTypes = {
     constraints: React.PropTypes.object,
   };
+
+  considerType(value) {
+    if (_.isString(value)) {
+      return `"${value}"`;
+    }
+    if (_.isNull(value)) {
+      return 'null';
+    }
+    return value;
+  }
 
   render() {
     const { constraints } = this.props;
@@ -35,9 +44,10 @@ class Constraints extends Component {
         }
 
         {constraints.get('enum') ?
-          <li>valid values: {constraints.get('enum').valueSeq().map( value =>
+          <li>valid values: {constraints.get('enum').valueSeq().map(value =>
             <code key={value}>{this.considerType(value)}</code>
-          ).reduce( (prev, curr) => [prev, ', ', curr] ) }</li>
+          )
+          .reduce((prev, curr) => [prev, ', ', curr])}</li>
         :
           constraints.get('type') === 'boolean' && <li>valid values: (true,false)</li>
         }
@@ -47,17 +57,6 @@ class Constraints extends Component {
         {constraints.get('notes') && <li>notes: {constraints.get('notes')}</li>}
       </ul>
     );
-  }
-
-  considerType (value) {
-
-    if(_.isString(value)){
-      return "\"" + value + "\"";
-    }
-    if(_.isNull(value)){
-      return "null";
-    }
-    return value;
   }
 
 }

